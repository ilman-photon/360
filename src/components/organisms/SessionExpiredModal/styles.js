export const styles = {
  buttonContainer: {
    display: "flex",
    flexDirection: "row",
    marginBottom: 16,
  },

  buttonStyle: {
    minWidth: "40px !important",
    marginLeft: "10px",
    fontSize: "14px",
    boxShadow: "none",
    transform: "none",
  },
  containerPadding: {
    padding: "16px 16px 0px 16px",
  },

  postMessageGenericError: {
    fontFamily: "Libre Franklin, sans-serif",
    fontStyle: "normal",
    fontWeight: "400",
    fontSize: "14px",
    lineHeight: "18px",
  },
};
