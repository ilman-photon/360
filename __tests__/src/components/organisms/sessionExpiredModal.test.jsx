import { fireEvent, render } from "@testing-library/react";
import "@testing-library/jest-dom";
import SessionExpiredModal from "../../../../src/components/organisms/SessionExpiredModal/sessionExpiredModal";
jest.mock("universal-cookie", () => {
  class MockCookies {
    static result = {};
    get() {
      return MockCookies.result;
    }
    remove() {
      return jest.fn();
    }
  }
  return MockCookies;
});

describe("SessionExpiredModal Components", () => {
  let container;
  beforeEach(() => {
    container = render(<SessionExpiredModal showModal={true} />);
  });
  afterAll(() => {
    jest.resetAllMocks();
  });
  it("render component", () => {
    expect(container.getByText("Session Timeout")).toBeInTheDocument();
  });
  it("render component expired true", () => {
    container = render(
      <SessionExpiredModal showModal={true} isExpired={true} />
    );
    expect(container.getByText("Session Timeout")).toBeInTheDocument();
  });
  it("render component expired true", () => {
    container = render(
      <SessionExpiredModal showModal={false} isExpired={true} />
    );
    expect(container.getByText("Session Timeout")).toBeInTheDocument();
  });
  it("render component show modal false", () => {
    container = render(<SessionExpiredModal showModal={false} />);
    expect(container.getByText("Session Timeout")).toBeInTheDocument();
  });
  it("click log off", () => {
    const logOffBtn = container.getByTestId("session-logoff-btn");
    fireEvent.click(logOffBtn);
  });
  it("click log in", () => {
    const logInBtn = container.getByTestId("session-stay-btn");
    fireEvent.click(logInBtn);
  });
});
