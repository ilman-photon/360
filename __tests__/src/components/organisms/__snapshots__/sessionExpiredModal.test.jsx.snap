// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SessionExpiredModal Components render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    style="padding-right: 1024px; overflow: hidden;"
  >
    <div
      aria-hidden="true"
    />
    <div
      class="MuiDialog-root MuiModal-root css-zw3mfo-MuiModal-root-MuiDialog-root"
      role="presentation"
    >
      <div
        aria-hidden="true"
        class="MuiBackdrop-root css-yiavyu-MuiBackdrop-root-MuiDialog-backdrop"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      />
      <div
        data-test="sentinelStart"
        tabindex="0"
      />
      <div
        class="MuiDialog-container MuiDialog-scrollPaper css-hz1bth-MuiDialog-container"
        role="presentation"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        tabindex="-1"
      >
        <div
          aria-describedby="alert-dialog-description"
          aria-labelledby="alert-dialog-title"
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation24 MuiDialog-paper MuiDialog-paperScrollPaper MuiDialog-paperWidthSm css-1t1j96h-MuiPaper-root-MuiDialog-paper"
          role="dialog"
        >
          <h2
            aria-label="Session Timeout heading"
            class="MuiTypography-root MuiTypography-h6 MuiDialogTitle-root css-bdhsul-MuiTypography-root-MuiDialogTitle-root"
            id="alert-dialog-title"
            style="padding: 16px 16px 0px 16px;"
          >
            Session Timeout
          </h2>
          <div
            class="MuiDialogContent-root css-14ywww5-MuiDialogContent-root"
            style="padding: 16px 16px 16px 16px;"
          >
            <div
              class="MuiBox-root css-1y4mj6z"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1jteomo-MuiSvgIcon-root"
                data-testid="BlockIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zM4 12c0-4.42 3.58-8 8-8 1.85 0 3.55.63 4.9 1.69L5.69 16.9C4.63 15.55 4 13.85 4 12zm8 8c-1.85 0-3.55-.63-4.9-1.69L18.31 7.1C19.37 8.45 20 10.15 20 12c0 4.42-3.58 8-8 8z"
                />
              </svg>
              <div
                aria-live="off"
                data-testid="submission-message"
                role="alert"
                style="color: rgb(255, 255, 255); font-size: 14px; line-height: 20.02px; letter-spacing: 0.17px;"
              >
                <div
                  style="font-size: 14px; font-weight: 400;"
                >
                  Your session is about to time-out. You will be logged out in 0 seconds.
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiDialogActions-root MuiDialogActions-spacing css-hlj6pa-MuiDialogActions-root"
            style="padding: 16px 16px 0px 16px;"
          >
            <div
              class="MuiBox-root css-0"
              style="display: flex; flex-direction: row; margin-bottom: 16px;"
            >
              <button
                aria-label="Log off button"
                class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root customButton customButtonSmall   sxButton css-19s7hx2-MuiButtonBase-root-MuiButton-root"
                data-testid="session-logoff-btn"
                style="min-width: 40px !important; margin-left: 10px; font-size: 14px; box-shadow: none; transform: none;"
                tabindex="0"
                type="button"
              >
                Log Off
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                aria-label="Stay Logged in button"
                class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root customButton customButtonSmall   sxButton css-1ai4zn7-MuiButtonBase-root-MuiButton-root"
                data-testid="session-stay-btn"
                style="min-width: 40px !important; margin-left: 10px; font-size: 14px; box-shadow: none; transform: none;"
                tabindex="0"
                type="button"
              >
                Stay Logged in
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        data-test="sentinelEnd"
        tabindex="0"
      />
    </div>
  </body>,
  "container": <div
    aria-hidden="true"
  />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`SessionExpiredModal Components render component expired true 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    style="padding-right: 1024px; overflow: hidden;"
  >
    <div
      aria-hidden="true"
    />
    <div
      aria-hidden="true"
      class="MuiDialog-root MuiModal-root css-zw3mfo-MuiModal-root-MuiDialog-root"
      role="presentation"
    >
      <div
        aria-hidden="true"
        class="MuiBackdrop-root css-yiavyu-MuiBackdrop-root-MuiDialog-backdrop"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      />
      <div
        data-test="sentinelStart"
        tabindex="0"
      />
      <div
        class="MuiDialog-container MuiDialog-scrollPaper css-hz1bth-MuiDialog-container"
        role="presentation"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        tabindex="-1"
      >
        <div
          aria-describedby="alert-dialog-description"
          aria-labelledby="alert-dialog-title"
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation24 MuiDialog-paper MuiDialog-paperScrollPaper MuiDialog-paperWidthSm css-1t1j96h-MuiPaper-root-MuiDialog-paper"
          role="dialog"
        >
          <h2
            aria-label="Session Timeout heading"
            class="MuiTypography-root MuiTypography-h6 MuiDialogTitle-root css-bdhsul-MuiTypography-root-MuiDialogTitle-root"
            id="alert-dialog-title"
            style="padding: 16px 16px 0px 16px;"
          >
            Session Timeout
          </h2>
          <div
            class="MuiDialogContent-root css-14ywww5-MuiDialogContent-root"
            style="padding: 16px 16px 16px 16px;"
          >
            <div
              class="MuiBox-root css-1y4mj6z"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1jteomo-MuiSvgIcon-root"
                data-testid="BlockIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zM4 12c0-4.42 3.58-8 8-8 1.85 0 3.55.63 4.9 1.69L5.69 16.9C4.63 15.55 4 13.85 4 12zm8 8c-1.85 0-3.55-.63-4.9-1.69L18.31 7.1C19.37 8.45 20 10.15 20 12c0 4.42-3.58 8-8 8z"
                />
              </svg>
              <div
                aria-live="off"
                data-testid="submission-message"
                role="alert"
                style="color: rgb(255, 255, 255); font-size: 14px; line-height: 20.02px; letter-spacing: 0.17px;"
              >
                <div
                  style="font-size: 14px; font-weight: 400;"
                >
                  Your session is about to time-out. You will be logged out in 0 seconds.
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiDialogActions-root MuiDialogActions-spacing css-hlj6pa-MuiDialogActions-root"
            style="padding: 16px 16px 0px 16px;"
          >
            <div
              class="MuiBox-root css-0"
              style="display: flex; flex-direction: row; margin-bottom: 16px;"
            >
              <button
                aria-label="Log off button"
                class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root customButton customButtonSmall   sxButton css-19s7hx2-MuiButtonBase-root-MuiButton-root"
                data-testid="session-logoff-btn"
                style="min-width: 40px !important; margin-left: 10px; font-size: 14px; box-shadow: none; transform: none;"
                tabindex="0"
                type="button"
              >
                Log Off
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                aria-label="Stay Logged in button"
                class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root customButton customButtonSmall   sxButton css-1ai4zn7-MuiButtonBase-root-MuiButton-root"
                data-testid="session-stay-btn"
                style="min-width: 40px !important; margin-left: 10px; font-size: 14px; box-shadow: none; transform: none;"
                tabindex="0"
                type="button"
              >
                Stay Logged in
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        data-test="sentinelEnd"
        tabindex="0"
      />
    </div>
    <div
      aria-hidden="true"
    />
    <div
      class="MuiDialog-root MuiModal-root css-zw3mfo-MuiModal-root-MuiDialog-root"
      role="presentation"
    >
      <div
        aria-hidden="true"
        class="MuiBackdrop-root css-yiavyu-MuiBackdrop-root-MuiDialog-backdrop"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      />
      <div
        data-test="sentinelStart"
        tabindex="0"
      />
      <div
        class="MuiDialog-container MuiDialog-scrollPaper css-hz1bth-MuiDialog-container"
        role="presentation"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        tabindex="-1"
      >
        <div
          aria-describedby="alert-dialog-description"
          aria-labelledby="alert-dialog-title"
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation24 MuiDialog-paper MuiDialog-paperScrollPaper MuiDialog-paperWidthSm css-1t1j96h-MuiPaper-root-MuiDialog-paper"
          role="dialog"
        >
          <h2
            aria-label="Session Expired heading"
            class="MuiTypography-root MuiTypography-h6 MuiDialogTitle-root css-bdhsul-MuiTypography-root-MuiDialogTitle-root"
            id="alert-dialog-title"
            style="padding: 16px 16px 0px 16px;"
          >
            Session Expired
          </h2>
          <div
            class="MuiDialogContent-root css-14ywww5-MuiDialogContent-root"
            style="padding: 16px 16px 16px 16px;"
          >
            <div
              class="MuiBox-root css-1y4mj6z"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1jteomo-MuiSvgIcon-root"
                data-testid="BlockIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zM4 12c0-4.42 3.58-8 8-8 1.85 0 3.55.63 4.9 1.69L5.69 16.9C4.63 15.55 4 13.85 4 12zm8 8c-1.85 0-3.55-.63-4.9-1.69L18.31 7.1C19.37 8.45 20 10.15 20 12c0 4.42-3.58 8-8 8z"
                />
              </svg>
              <div
                aria-live="off"
                data-testid="submission-message"
                role="alert"
                style="color: rgb(255, 255, 255); font-size: 14px; line-height: 20.02px; letter-spacing: 0.17px;"
              >
                <div
                  style="font-size: 14px; font-weight: 400;"
                >
                  Your session expired. Please login again.
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiDialogActions-root MuiDialogActions-spacing css-hlj6pa-MuiDialogActions-root"
            style="padding: 16px 16px 0px 16px;"
          >
            <div
              class="MuiBox-root css-0"
              style="display: flex; flex-direction: row; margin-bottom: 16px;"
            >
              <button
                aria-label="Ok button"
                class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root customButton customButtonSmall   sxButton css-1ai4zn7-MuiButtonBase-root-MuiButton-root"
                data-testid="session-ok-btn"
                style="min-width: 40px !important; margin-left: 10px; font-size: 14px; box-shadow: none; transform: none;"
                tabindex="0"
                type="button"
              >
                OK
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        data-test="sentinelEnd"
        tabindex="0"
      />
    </div>
  </body>,
  "container": <div
    aria-hidden="true"
  />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`SessionExpiredModal Components render component expired true 2`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    style="padding-right: 1024px; overflow: hidden;"
  >
    <div
      aria-hidden="true"
    />
    <div
      class="MuiDialog-root MuiModal-root css-zw3mfo-MuiModal-root-MuiDialog-root"
      role="presentation"
    >
      <div
        aria-hidden="true"
        class="MuiBackdrop-root css-yiavyu-MuiBackdrop-root-MuiDialog-backdrop"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      />
      <div
        data-test="sentinelStart"
        tabindex="0"
      />
      <div
        class="MuiDialog-container MuiDialog-scrollPaper css-hz1bth-MuiDialog-container"
        role="presentation"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        tabindex="-1"
      >
        <div
          aria-describedby="alert-dialog-description"
          aria-labelledby="alert-dialog-title"
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation24 MuiDialog-paper MuiDialog-paperScrollPaper MuiDialog-paperWidthSm css-1t1j96h-MuiPaper-root-MuiDialog-paper"
          role="dialog"
        >
          <h2
            aria-label="Session Timeout heading"
            class="MuiTypography-root MuiTypography-h6 MuiDialogTitle-root css-bdhsul-MuiTypography-root-MuiDialogTitle-root"
            id="alert-dialog-title"
            style="padding: 16px 16px 0px 16px;"
          >
            Session Timeout
          </h2>
          <div
            class="MuiDialogContent-root css-14ywww5-MuiDialogContent-root"
            style="padding: 16px 16px 16px 16px;"
          >
            <div
              class="MuiBox-root css-1y4mj6z"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1jteomo-MuiSvgIcon-root"
                data-testid="BlockIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zM4 12c0-4.42 3.58-8 8-8 1.85 0 3.55.63 4.9 1.69L5.69 16.9C4.63 15.55 4 13.85 4 12zm8 8c-1.85 0-3.55-.63-4.9-1.69L18.31 7.1C19.37 8.45 20 10.15 20 12c0 4.42-3.58 8-8 8z"
                />
              </svg>
              <div
                aria-live="off"
                data-testid="submission-message"
                role="alert"
                style="color: rgb(255, 255, 255); font-size: 14px; line-height: 20.02px; letter-spacing: 0.17px;"
              >
                <div
                  style="font-size: 14px; font-weight: 400;"
                >
                  Your session is about to time-out. You will be logged out in 0 seconds.
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiDialogActions-root MuiDialogActions-spacing css-hlj6pa-MuiDialogActions-root"
            style="padding: 16px 16px 0px 16px;"
          >
            <div
              class="MuiBox-root css-0"
              style="display: flex; flex-direction: row; margin-bottom: 16px;"
            >
              <button
                aria-label="Log off button"
                class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root customButton customButtonSmall   sxButton css-19s7hx2-MuiButtonBase-root-MuiButton-root"
                data-testid="session-logoff-btn"
                style="min-width: 40px !important; margin-left: 10px; font-size: 14px; box-shadow: none; transform: none;"
                tabindex="0"
                type="button"
              >
                Log Off
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                aria-label="Stay Logged in button"
                class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root customButton customButtonSmall   sxButton css-1ai4zn7-MuiButtonBase-root-MuiButton-root"
                data-testid="session-stay-btn"
                style="min-width: 40px !important; margin-left: 10px; font-size: 14px; box-shadow: none; transform: none;"
                tabindex="0"
                type="button"
              >
                Stay Logged in
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        data-test="sentinelEnd"
        tabindex="0"
      />
    </div>
    <div />
  </body>,
  "container": <div />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`SessionExpiredModal Components render component show modal false 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    style="padding-right: 1024px; overflow: hidden;"
  >
    <div
      aria-hidden="true"
    />
    <div
      class="MuiDialog-root MuiModal-root css-zw3mfo-MuiModal-root-MuiDialog-root"
      role="presentation"
    >
      <div
        aria-hidden="true"
        class="MuiBackdrop-root css-yiavyu-MuiBackdrop-root-MuiDialog-backdrop"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      />
      <div
        data-test="sentinelStart"
        tabindex="0"
      />
      <div
        class="MuiDialog-container MuiDialog-scrollPaper css-hz1bth-MuiDialog-container"
        role="presentation"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        tabindex="-1"
      >
        <div
          aria-describedby="alert-dialog-description"
          aria-labelledby="alert-dialog-title"
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation24 MuiDialog-paper MuiDialog-paperScrollPaper MuiDialog-paperWidthSm css-1t1j96h-MuiPaper-root-MuiDialog-paper"
          role="dialog"
        >
          <h2
            aria-label="Session Timeout heading"
            class="MuiTypography-root MuiTypography-h6 MuiDialogTitle-root css-bdhsul-MuiTypography-root-MuiDialogTitle-root"
            id="alert-dialog-title"
            style="padding: 16px 16px 0px 16px;"
          >
            Session Timeout
          </h2>
          <div
            class="MuiDialogContent-root css-14ywww5-MuiDialogContent-root"
            style="padding: 16px 16px 16px 16px;"
          >
            <div
              class="MuiBox-root css-1y4mj6z"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1jteomo-MuiSvgIcon-root"
                data-testid="BlockIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zM4 12c0-4.42 3.58-8 8-8 1.85 0 3.55.63 4.9 1.69L5.69 16.9C4.63 15.55 4 13.85 4 12zm8 8c-1.85 0-3.55-.63-4.9-1.69L18.31 7.1C19.37 8.45 20 10.15 20 12c0 4.42-3.58 8-8 8z"
                />
              </svg>
              <div
                aria-live="off"
                data-testid="submission-message"
                role="alert"
                style="color: rgb(255, 255, 255); font-size: 14px; line-height: 20.02px; letter-spacing: 0.17px;"
              >
                <div
                  style="font-size: 14px; font-weight: 400;"
                >
                  Your session is about to time-out. You will be logged out in 0 seconds.
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiDialogActions-root MuiDialogActions-spacing css-hlj6pa-MuiDialogActions-root"
            style="padding: 16px 16px 0px 16px;"
          >
            <div
              class="MuiBox-root css-0"
              style="display: flex; flex-direction: row; margin-bottom: 16px;"
            >
              <button
                aria-label="Log off button"
                class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root customButton customButtonSmall   sxButton css-19s7hx2-MuiButtonBase-root-MuiButton-root"
                data-testid="session-logoff-btn"
                style="min-width: 40px !important; margin-left: 10px; font-size: 14px; box-shadow: none; transform: none;"
                tabindex="0"
                type="button"
              >
                Log Off
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                aria-label="Stay Logged in button"
                class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root customButton customButtonSmall   sxButton css-1ai4zn7-MuiButtonBase-root-MuiButton-root"
                data-testid="session-stay-btn"
                style="min-width: 40px !important; margin-left: 10px; font-size: 14px; box-shadow: none; transform: none;"
                tabindex="0"
                type="button"
              >
                Stay Logged in
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        data-test="sentinelEnd"
        tabindex="0"
      />
    </div>
    <div />
  </body>,
  "container": <div />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
